docs/src/tutorial/genomic-surveillance.rst

```diff
@@ -0,0 +1,158 @@
+Run the workflow using a genomic surveillance configuration
+===========================================================
+
+In this tutorial, you will replace the custom contextual data with meaningful focal data and use a configuration commonly found in genomic surveillance applications.
+
+.. contents:: Table of Contents
+   :local:
+
+Prerequisites
+-------------
+
+1. :doc:`custom-data`. These instructions will introduce the workflow configuration that is modified in this tutorial.
+
+Setup
+-----
+
+If you are not already there, change directory to the ``ncov`` directory:
+
+   .. code:: text
+
+      cd ncov
+
+Curate data from GISAID
+-----------------------
+
+.. warning::
+
+   These steps require a GISAID account. `Register <https://www.gisaid.org/registration/register/>`__ if you do not have an account yet. However, registration may take a few days. Follow `alternative data preparation methods <../guides/data-prep.html>`__ if you wish to continue this tutorial in the meantime.
```

Comment:
Same comment as the "custom data" tutorial: this should probably be in the Setup or at least listed as a prerequisite.

Reply:
Done: 563598af